### PR TITLE
Isolated state manager and modified replicaCount values

### PR DESCRIPTION
**Pull request by nadinequinr3**
Preview: https://nadine.preview.docs.r3.com/en/platform/corda/5.1.html

### CODE REVIEW

File: content/en/platform/corda/5.1/deploying-operating/deployment/deploying/_index.md [hunk 1]
```diff
@@ -762,19 +888,15 @@ bootstrap:
     enabled: false
 ```
 
-If the pre-install checks fail, for example, if Kafka or PostgreSQL are not available, you can retrieve the logs for a
-single pod using `kubectl`. For more information on retrieving the logs, see the [Logs]({{< relref "../../observability/logs.md" >}}) section.
+If the pre-install checks fail, for example, if Kafka or PostgreSQL are not available, you can retrieve the logs for a single pod using `kubectl`. For more information on retrieving the logs, see the [Logs]({{< relref "../../observability/logs.md" >}}) section.
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.1/deploying-operating/deployment/deploying/_index.md:891]:
```suggestion
If the pre-install checks fail (for example, if Kafka or PostgreSQL are not available), you can retrieve the logs for a single pod using `kubectl`. For more information on retrieving the logs, see the [Logs]({{< relref "../../observability/logs.md" >}}) section.
```

**Review by paulmoloneyr3 — CHANGES_REQUESTED** [review]

**Review by paulmoloneyr3 — DISMISSED** [review]

File: content/en/platform/corda/5.1/deploying-operating/deployment/deploying/_index.md [hunk 2]
```diff
@@ -404,6 +418,87 @@ db:
     database: <POSTGRESQL_CLUSTER_DB>
 ```
 
+#### State Manager Databases
+
+Corda requires one or more PostgreSQL database instances for the persistence of state between worker types. These are referred to as state manager databases. R3 recommends that you deploy separate isolated state manager database instances for the following workers:
```
Comment (conalsmith-r3) [inline content/en/platform/corda/5.1/deploying-operating/deployment/deploying/_index.md:423]: `persistence of state between worker types` isn't quite correct, it probably should be 

"Corda requires one or more PostgreSQL database instances for the persistence of state."

File: content/en/platform/corda/5.1/deploying-operating/deployment/deploying/_index.md [hunk 3]
```diff
@@ -404,6 +418,87 @@ db:
...
+
+* Flow workers
+* Flow mapper workers
+* Token selection workers
+
+{{< note >}}
+If you do not configure state manager databases, Corda deploys the state managers for the flow, flow mapper, and token selection workers on the cluster database. This is not safe for production deployments.
```
[the remark this answers is not in the document]
Comment (nadinequinr3) [reply content/en/platform/corda/5.1/deploying-operating/deployment/deploying/_index.md:428]: So do we need a section before https://nadine.preview.docs.r3.com/en/platform/corda/5.1/deploying-operating/deployment/deploying/manual-bootstrapping.html#create-the-database-tables to say that you must manually create separate state manager database instances for the 3 worker types?

[the remark this answers is not in the document]
Comment (conalsmith-r3) [reply content/en/platform/corda/5.1/deploying-operating/deployment/deploying/_index.md:428]: Well we already say just above it:

```
Corda requires one or more PostgreSQL database instances for the persistence of state. These are referred to as state manager databases. R3 recommends that you deploy separate isolated state manager database instances for the following workers:
```

And in manual bootstrapping we say:

```
Review the DML file generated and execute it against each of the State Manager databases.
```

Maybe all we need to say is, 
```
If you do not configure state manager databases and you have auto-bootstrapping enabled, Corda helm installation will deploy the state managers for the flow, flow mapper, and token selection workers on the cluster database. This is not safe for production deployments.
```

[the remark this answers is not in the document]
Comment (conalsmith-r3) [reply content/en/platform/corda/5.1/deploying-operating/deployment/deploying/_index.md]: Let's leave it as is, as you say this whole page assumes auto-bootstrapping is used

[the remark this answers is not in the document]
Comment (nadinequinr3) [reply content/en/platform/corda/5.1/deploying-operating/deployment/deploying/_index.md:428]: Done

**Review by paulmoloneyr3 — APPROVED** [review]

**Review by conalsmith-r3 — APPROVED** [review]
lgtm

**Pull request merged**